src/renderer/components/delete-cluster-dialog/delete-cluster-dialog.tsx

```diff
@@ -52,15 +52,12 @@ type Props = {};
 
 @observer
 export class DeleteClusterDialog extends React.Component {
-  showContextSwitch = false;
-  newCurrentContext = "";
+  @observable showContextSwitch = false;
```

Comment:
This file changes fixes a new error from `mobx`:

```
fix: mobx.esm.js:87 Uncaught Error: [MobX] makeObservable second arg … …
e69655b
…must be nullish when using decorators. Mixing @decorator syntax with annotations is not supported.
```

So, it seems like from this PR we have to stick to single style of annotations within a class? 
Use decorators or not (_yes_).

Reply:
I think we use decorators more often so at least for the time being we should stick with that majority.

PR looks good, just going to test it and then approve.